src/app.js

```diff
@@ -10,13 +10,9 @@ import './styles/styles.scss';
 
 const store = configureStore();
 
-// const state = () => console.log(store.getState());
-// store.subscribe(state);
-const entryOne = store.dispatch(addActivity({name: 'matma', day: '2', classNo: 3, room: 'gym'}));
+//dummy data to be removed
```

Comment:
I would add `TODO` notification. In thhe future you will be able to search all todos and resolve them. Anyway, I am giving approval, you can merge (after adding todo and pushing :P)